option optimize_for = LITE_RUNTIME;

message LegionChangeAcceptTypeAnswer
{
 required uint32 result = 1;
 required uint32 accept_type = 2;
 required uint32 accept_value = 3;
}
